release: 0.9.0 (#3134)
rome/tools: website/src/_includes/docs/architecture.md

## Architecture (in progress)

Rome uses a server-client architecture to run its tasks.

### Daemon

A [daemon](https://en.wikipedia.org/wiki/Daemon_(computing)) is a long-running server 

> that Rome will spawn in the background, and will use it to process a series tasks as it sees fit.

Reviewer: ```suggestion
that Rome spawns in the background and uses to process requests from the editor and CLI. 
```
